cyences_app_for_splunk/default/macros.conf

```diff
@@ -729,6 +751,10 @@ iseval = 0
 definition = index=pan_log
 iseval = 0
 
+[cs_palo_commit_filter]
```

Comment:
I forgot to mention for each macro that we create we need to update the `cs_configuration.xml` and `cs_configuration.js` files.

Can you please also update for last PRs that we merge recently? Sorry I missed that in the last PRs.